docs/faq.md

```diff
@@ -21,3 +21,21 @@ Please check the following list.
 1. Add a user from [Django Admin site](https://djangobook.com/django-admin-site/).
 ![Add a user](./images/faq/add_user.png)
 2. Add the user to the project in the member page(`/projects/{project_id}/members`).
+
+## I want to rebuild my doccano with latest changes
```

Comment:
```suggestion
## I want to rebuild my Doccano with latest changes
```

Reply:
**d**occano is the small case. No fix is needed

Reply:
Updated.